docs/getting-started-on-snowplow-bdp-cloud/first-party-tracking/index.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "First party tracking"
+sidebar_position: 3
+sidebar_custom_props:
+  offerings:
+    - cloud
+---
+
+# First party tracking
+
+This feature lets you configure a custom collector domain to match the domain you wish to track. This allows for first-party cookies to be set on the server.
+Enter the domain you wish to track, **any sub domain or multi level sub domain will be removed** and a cookie domain will be generated. If you wish to restrict the tracking to a sub domain the cookie domain can still be edited.
+
+![enter_domain](images/Screenshot_enter_domain.png)
+
+## DNS records
+The required DNS records will now be created, this may take several minutes.
+When the records are ready you will receive email confirmation.
```

Comment:
```suggestion
BDP Cloud will now generate the required DNS records. This may take several minutes.
When the records are ready, you will receive a confirmation by email.
```
Which email? The one used to sign up?